src/modules/endpoint/sagas/watchContentLanguageSwitch.js

```diff
@@ -47,6 +52,9 @@ export function * switchContentLanguage (dataContainer: DataContainer, action: S
     }
     yield put(insert)
   } catch (e) {
+    if (e.message === 'Network request failed') {
```

Comment:
We should add here a note into some issue to change this. I added it here: https://issues.integreat-app.de/browse/NATIVE-354